vaccine_feed_ingest/runners/ri/prepmod/fetch.yml

```diff
@@ -1,5 +1,5 @@
 ---
 state: ri
-site: prepmod
+site: ri_prepmod
```

Comment:
This will change the ids for all of these locations, right? If so then it is too late to make this change.

Reply:
yeah :/ thats why i was concerned about how breaking this change was.

on one hand if im combining the duplicate prepmod locations as part of #814 the ID's will change anyway, on the other hand, if this is the only scraper with the generic `prepmod` id, then theres no confusion and this issue is more pedantic than an actual problem